scripts/src/Platform/MacOS.hs

```diff
@@ -87,8 +90,8 @@ isLocalDep dep = categorizeDependency dep == Local
 installBinary :: (MonadIO m) => FilePath -> FilePath -> m FilePath
 installBinary targetBinariesDir sourcePath = do
     -- putStrLn $ "installing " <> takeFileName sourcePath <> " to " <> targetBinariesDir
-    destinationPath <- copyToDir targetBinariesDir sourcePath
-    runProcess_ $ proc "chmod" ["777", destinationPath]
+    destinationPath <- Utils.copyToDir targetBinariesDir sourcePath
+    Process.runProcess_ $ Process.proc "chmod" ["777", destinationPath]
```

Comment:
For those unfamiliar with `chmod` it might be worth including a comment as to what permissions `777` means.